packages/WebVideo/WebVideo.jsx

```diff
@@ -105,7 +110,7 @@ WebVideo.propTypes = {
   /**
    * The aspect ratio of the player.
    */
-  aspectRatio: PropTypes.oneOf(['16:9', '4:3', '1:1']),
```

Comment:
Let's keep these options available to users, while also allowing custom aspect ratios.